angular/test/test-app/tslint.json

```diff
@@ -1,36 +1,16 @@
 {
-  "rulesDirectory": [
-    "node_modules/codelyzer"
-  ],
+  "extends": "tslint:recommended",
+  "rulesDirectory": ["codelyzer"],
   "rules": {
-    "arrow-return-shorthand": true,
```

Comment:
why all this changes?

Reply:
ng8 updates

Reply:
as part of the ng8 update, codelyzer also released a 5.0 which included some rules being changed/renamed. This just followed the standard ng8 app starter.